essential-projects/core_contracts
# Split content of interface.ts

Closes #4 

## What did you change?

- All interfaces, enums, asf. are now located in separate files.
- Interfaces and enums have been grouped into UseCase-specific folders (where applicable).

## How can others test the changes?

- Look through the source folder, you'll immediately see the changes.
- Build a solution that uses the core_contracts project. The build should run exactly as before, since all changes made were exclusively cosmetic and non-functional. 

## PR-Checklist

Please check the boxes in this list after submitting your PR:
